x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -234,12 +235,16 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
         version: this.context.env.packageInfo.version,
         ml: plugins.ml,
         lists: plugins.lists,
+        errorService: core.getStartServices().then(async ([coreStart]) => {
+          return coreStart.savedObjects.createInternalRepository();
+        }),
       });
       const ruleNotificationType = rulesNotificationAlertType({
         logger: this.logger,
       });
 
       if (isAlertExecutor(signalRuleType)) {
+        // add status callback here in order to pick up failures on task manager
```

Comment:
nit: this looks like just an extra comment? Is this comment placed correctly?

Reply:
yup that should be removed. Thanks!